packages/esm-patient-allergies-app/src/allergies/allergies-tile.component.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useAllergies } from './allergy-intolerance.resource';
+import { useTranslation } from 'react-i18next';
+import styles from './allergies-tile.component.scss';
+
+interface AllergyTileInterface {
+  patientUuid: string;
+}
+
+const AllergyTile: React.FC<AllergyTileInterface> = ({ patientUuid }) => {
+  const { data: allergies } = useAllergies(patientUuid);
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.gridColumn}>
+      <p>{t('allergies', 'Allergies')}</p>
+      <span>{allergies?.map((allergy) => allergy?.display).join(', ')}</span>
```

Comment:
Can `allergy` actually be falsey here (where you're using `allergy?.display`)? Because if so you will need to `filter` after the `map` so that you don't get `, ,`s.